local-test-infra.sh

```diff
@@ -0,0 +1,19 @@
+mkdir -p ./temp
+
+export res=$(oc get pod -A -l test-network-function.com/generic=fs_diff_master 2>&1)
+if [ "$res" == "No resources found" ]; then
+    cat $TNF_PARTNER_SRC_DIR/local-test-infra/fsdiff-pod.yaml | ./script/mo > ./temp/rendered-fsdiff-template.yaml
```

Comment:
Why not run "make clean" and "make install" in the partner repo? Usually apply handles already running pods and the creation logic is already implemented in the partner repo. We could also use mo only in the partner repo.